Mathlib/Analysis/PSeries.lean

```diff
@@ -255,34 +261,43 @@ theorem Real.not_summable_one_div_nat_cast : ¬Summable (fun n => 1 / n : ℕ 
 theorem Real.tendsto_sum_range_one_div_nat_succ_atTop :
     Tendsto (fun n => ∑ i in Finset.range n, (1 / (i + 1) : ℝ)) atTop atTop := by
   rw [← not_summable_iff_tendsto_nat_atTop_of_nonneg]
-  · exact_mod_cast mt (summable_nat_add_iff 1).1 Real.not_summable_one_div_nat_cast
+  · exact_mod_cast mt (_root_.summable_nat_add_iff 1).1 Real.not_summable_one_div_nat_cast
   · exact fun i => div_nonneg zero_le_one i.cast_add_one_pos.le
 #align real.tendsto_sum_range_one_div_nat_succ_at_top Real.tendsto_sum_range_one_div_nat_succ_atTop
 
+section pow_macro
+-- porting note: without these local macro rules Lean fails to elaborate `^` properly.
+-- See https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/!4.234085
+local macro_rules | `($x ^ $y)   => `(HPow.hPow $x $y)
```

Comment:
@digama0 Do you understand what the correct solution is here?

Reply:
Aha: this work-around has been discussed here https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/!4.234085/near/361005815 .